installer/module/streamlit.py

```
@@ -70,6 +70,67 @@ def fetch_streamlit_apps(logger: logging.Logger) -> List[dict]:
+    logger.info(f"Fetching integration environment variables from {url}.")
+
+    try:
+        json_content = request_with_retries(
+            logger,
+            url,
+            max_retries=max_retries,
+            timeout=timeout,
+        )
+        variables = json.loads(json_content)
+        logger.info(
+            f"Fetched {len(variables)} integration environment variables."
```

Ruff (0.15.13)

[warning] Logging statement uses f-string

(G004)